src/mongo/Driver.class.php

```diff
@@ -180,11 +180,12 @@ public function getViewForResources(Array $resources, $viewType)
     /**
      * @param array $filter
      * @param string $viewType
+     * @param string $readPreference
      * @return MongoGraph
      */
-    public function getViews(Array $filter, $viewType)
+    public function getViews(Array $filter, $viewType, $readPreference = \MongoClient::RP_PRIMARY_PREFERRED)
```

Comment:
I think this should default to `null` and then in the method:
```php 
if(is_null($readPreference))
{
    $readPreference = $this->readPreference;
}
```
I think it's counterintuitive if the developer has already set the default read preference when instantiating Tripod that the delegates wouldn't also respect that.